### PR TITLE
Rename 'macOS-11.0' to 'macOS-11' everywhere

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
`macOS-11.0` is not correct naming because currently, we are delivering macOS 11.3.
We should use `macOS-11` to not confuse customers

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

File: images/macos/macos-11-Readme.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+| Announcements |
+|-|
+| [macOS 11.0 pools will be transited to private preview.](https://github.com/actions/virtual-environments/issues/2486) |
```
Comment (miketimofeev) [inline images/macos/macos-11-Readme.md:3]: Should we fix this announcement as well? 😃

**Review by AlenaSviridenko — DISMISSED** [review]

File: images/macos/macos-11-Readme.md (hunk 1 above)
Comment (AlenaSviridenko) [reply images/macos/macos-11-Readme.md:3]: I think we should leave it as-is, do not see any sense in changing it

File: .github/ISSUE_TEMPLATE/announcement.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ assignees: ''
 - [ ] macOS 10.13
 - [ ] macOS 10.14
 - [ ] macOS 10.15
-- [ ] macOS 11.0
+- [ ] macOS 11
```
Comment (MaksimZhukov) [inline .github/ISSUE_TEMPLATE/announcement.md:29]: Do we need to rename it in the [bug-report.md](https://github.com/actions/virtual-environments/blob/main/.github/ISSUE_TEMPLATE/bug-report.md) and [tool-request.md](https://github.com/actions/virtual-environments/blob/main/.github/ISSUE_TEMPLATE/tool-request.md) as well?

**Review by MaksimZhukov — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/announcement.md (hunk 2 above)
Comment (maxim-lobanov) [reply .github/ISSUE_TEMPLATE/announcement.md:29]: Done

**Comment by ONFU** [comment]
the manifest these not availaible not able thow distributed by the rainbow helpshield of concuranty of ethernal tear of sorrow lt's comunicated by the NOX-PACT the districk philipine am general of the NoX-PacT and am valkhymher valentine valkirye vendetta my named it's kévin paré

**Pull request merged**